src/com/company/GameManager.java

```diff
@@ -22,39 +22,34 @@
     public static void play() {
 
         boolean shouldPlayAgain = true;
-        Player tableOfPlayers[] = new Player[NUMBER_OF_PLAYERS];
-        PlayerSignType sign = PlayerSignType.O;
+        final Player players[] = new Player[NUMBER_OF_PLAYERS];
+        final PlayerSignType sign[] = new PlayerSignType[NUMBER_OF_PLAYERS];
+
+        if (getRandomPlayerIndex(NUMBER_OF_PLAYERS) == 0) {
```

Comment:
Heh, wygląda strasznie.
Wyciągnij to do osobnej metody zwracającej tę tablicę a na najbliższej lekcji to przerobimy na listę tak żeby było "flexible and generic"

Reply:
zrobiłam osobną metodę kopiując tego if. Napisałam już gdzieś że widzę że to nie jest idealne rozwiązanie ale na tą chwilę nie widzę innego, szczególnie flexible. Ale z drugiej strony nie wiem co tu jest straszne ... ;)

Reply:
To na lekcji przerobimy, nie ma co się tutaj rozwodzić.